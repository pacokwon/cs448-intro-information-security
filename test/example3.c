#include "homework.h"

int main() {
  int i;
  int sum = 0;
  int z;
  for (i = 10; i > 1; i--) {
    sum += i;
  }
  z = 10 / (i - 1); // safe by interval with narrowing, false alarm by sign
  z = 10 / i;       // error
  return 0;
}
