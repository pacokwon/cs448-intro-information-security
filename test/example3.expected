Divide-by-zero detected at line 4 and col 15
